### DISCUSSION TITLE
Maintaining language variants

### DISCUSSION THREAD

**Discussion by docwarems** [intlify/vue-i18n#991] [Q&A]
Hi. from what I read from the docs it is not clear to me how the translation for a language variant is maintained. Say I would like to suppport swiss german (Locale de_CH), and only a few texts are different from default german (de_DE). Would I be able to maintain only the differences, or would I have to maintain a complete text set for the language variant?

**Comment by kazupon** [comment]
Unfortunately, it's not possible to keep only differentials currently.
However, it's possible to copy the target locale completely and apply the differences using `Object.assign`, etc.
